.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,11 +2,5 @@
 
 **JIRA:** Link to JIRA ticket
 
-**Merge checklist:**
-- [ ] All reviewers approved
```

Comment:
Agreed this isn't necessary *if the repo* is configured to force a review before merge.  This one does not force having a review.  Should it?